ansible/www-standalone/resources/config/nodejs.org

```diff
@@ -368,7 +368,8 @@ server {
     rewrite ^/(20\d\d/\d\d/\d\d/.*)$                              http://blog.nodejs.org/$1 permanent;
 
     rewrite ^/about/?$                                            https://$server_name/en/about/ permanent;
-    rewrite ^/about/releases/?$                                   https://$server_name/en/about/releases/ permanent;
+    rewrite ^/about/releases/?$                                   https://github.com/nodejs/release#release-schedule permanent;
+    rewrite ^/en/about/releases/?$                                https://github.com/nodejs/release#release-schedule permanent;
```

Comment:
is there a why to catch the i18n versions too, instead of just /en/

Reply:
I looked through the rest of the config file and saw lots of `en` redirects but nothing that looked like it would catch other languages. I agree it would be a good thing if that were a feature in this.....

Reply:
I think most of the ones in here were for before we had any i18n stuff in the original structure. I don't know the nginx syntax, but my basic guess would be some sort of regex like `rewrite ^/*/about/releases/?$`

Reply:
That's my gut too, but I'd want to consult the docs. Since there are already a bunch of `en`-specific redirects in the file, I'm inclined to consider "work for other languages" to be an enhancement that can go in a separate pull request. (Feel free to open it if you feel pretty sure about the syntax!)

Reply:
are you sure the `#` is going to be happy in here unescaped? might be worth double-checking that's OK